@here/harp-mapview/lib/MapView.ts

```diff
@@ -1818,25 +1870,39 @@ export class MapView extends THREE.EventDispatcher {
         this.m_camera.updateProjectionMatrix();
         this.m_camera.updateMatrixWorld(false);
 
+        this.m_rtcCamera.copy(this.m_camera);
+        this.m_rtcCamera.position.sub(this.m_camera.position);
```

Comment:
you are setting the camera to 0,0,0 right? I think setScalar(0) is a bit more clear what you are meaning.